pkgs/development/python-modules/splunk-sdk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "splunk-sdk";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "splunk";
+    repo = "splunk-sdk-python";
+    rev = "refs/tags/${version}";
+    hash = "sha256-N+QQ4DSkx7yakROhcJ2ISXPWFa7BXDeSUULNquhDPrg=";
+  };
+
+  build-system = [ setuptools ];
```

```suggestion
  nativeBuildInputs = [
    pypaInstallHook
    setuptoolsBuildHook
  ];
```
Setting `pyproject = false` and using this would be more *technically* correct, but they work the same. Up to you

---

> more *technically* correct

What does this mean?
If it means that there is no pyproject.toml, it is correct without changing it because it is specified to fall back to using setuptools as the build-system.

https://setuptools.pypa.io/en/latest/build_meta.html#what-is-it
> Note that if there is no pyproject.toml file or the build-backend parameter is not defined, then the fall-back behaviour is to use setuptools.

---

Huh TIL. I thought the fallback was only in Nixpkgs, not a part of PEP 517. Apologies